<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>479</width>
    <height>705</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Start/Stop EtherCAT server</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnStartServer">
          <property name="text">
           <string>START</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnServoON">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(224, 27, 36);</string>
        </property>
        <property name="text">
         <string>ServoOFF</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_2">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>State Observation - Axis 1</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QTableWidget" name="tblReadPDO">
         <property name="enabled">
          <bool>true</bool>
         </property>
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>437</width>
           <height>261</height>
          </size>
         </property>
         <property name="mouseTracking">
          <bool>false</bool>
         </property>
         <property name="tabletTracking">
          <bool>false</bool>
         </property>
         <property name="focusPolicy">
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="contextMenuPolicy">
          <enum>Qt::NoContextMenu</enum>
         </property>
         <property name="verticalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="horizontalScrollBarPolicy">
          <enum>Qt::ScrollBarAlwaysOff</enum>
         </property>
         <property name="autoScroll">
          <bool>false</bool>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="tabKeyNavigation">
          <bool>false</bool>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropOverwriteMode">
          <bool>false</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::NoSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectItems</enum>
         </property>
         <property name="horizontalScrollMode">
          <enum>QAbstractItemView::ScrollPerItem</enum>
         </property>
         <property name="showGrid">
          <bool>true</bool>
         </property>
         <property name="wordWrap">
          <bool>false</bool>
         </property>
         <property name="cornerButtonEnabled">
          <bool>false</bool>
         </property>
         <attribute name="horizontalHeaderVisible">
          <bool>true</bool>
         </attribute>
         <attribute name="horizontalHeaderCascadingSectionResizes">
          <bool>false</bool>
         </attribute>
         <attribute name="horizontalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <attribute name="verticalHeaderHighlightSections">
          <bool>true</bool>
         </attribute>
         <row>
          <property name="text">
           <string>Status Word (-)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Actual Position (Unit)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Actual Velocity (Unit/s)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Actual Toraqe (0.1%)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Position Command Value (Unit)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Velocity Command Value (Unit/s)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Torque Command Value (0.1%)</string>
          </property>
         </row>
         <row>
          <property name="text">
           <string>Error Code (-)</string>
          </property>
         </row>
         <column>
          <property name="text">
           <string notr="true">Adress</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Value</string>
          </property>
         </column>
         <item row="0" column="0">
          <property name="text">
           <string>0x6041 - 00h</string>
          </property>
         </item>
         <item row="1" column="0">
          <property name="text">
           <string>0x60B0 - 00h</string>
          </property>
         </item>
         <item row="2" column="0">
          <property name="text">
           <string>0x606C - 00h</string>
          </property>
         </item>
         <item row="3" column="0">
          <property name="text">
           <string>0x607A - 00h</string>
          </property>
         </item>
         <item row="4" column="0">
          <property name="text">
           <string>0x6062 - 00h</string>
          </property>
         </item>
         <item row="5" column="0">
          <property name="text">
           <string>0x606B - 00h</string>
          </property>
         </item>
         <item row="6" column="0">
          <property name="text">
           <string>0x6074 - 00h</string>
          </property>
         </item>
         <item row="7" column="0">
          <property name="text">
           <string>0x603F - 00h</string>
          </property>
         </item>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string>WritePDO</string>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLabel" name="lblOperationMode">
             <property name="text">
              <string>Operation mode</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblMaxMotorVelocity">
             <property name="text">
              <string>Max motor velocity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblTargetPositon">
             <property name="text">
              <string>Target position</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblTargetVelocity">
             <property name="text">
              <string>Target velocity</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblProfileAcceleration">
             <property name="text">
              <string>Profile acceleration</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="lblProfileDeceleration">
             <property name="text">
              <string>Profile deceleration</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QLineEdit" name="lineOperationMode"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineMaxMotorVelocity"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineTargetPosition"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineTargetVelocity"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineProfileAcceleration"/>
           </item>
           <item>
            <widget class="QLineEdit" name="lineProfileDeceleration"/>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QPushButton" name="btnOperationModeWrite">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnMaxMotorVelocity">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTargetPositionWrite">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnTargetVelocityWrite">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnAccelerationWrite">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="btnDecelerationWrite">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Write</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QPushButton" name="btnQuickStop">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>Quick stop</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnReset">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>RESET</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>479</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
